api-admin/src/test/java/bio/terra/pearl/api/admin/service/study/StudyExtServiceTests.java

```diff
@@ -115,4 +115,36 @@ public void testStudyDeletionNeedsSuperUser(TestInfo testInfo) {
     // confirm that the corresponding portalService also deleted
     assertThat(portalStudyService.findByPortalId(portal.getId()), not(empty()));
   }
+
+  @Test
+  @Transactional
+  void testCreateWithTemplate(TestInfo info) {
+    Portal portal = portalFactory.buildPersisted(getTestName(info));
+    for (EnvironmentName envName : EnvironmentName.values()) {
+      environmentFactory.buildPersisted(getTestName(info), envName);
+      portalEnvironmentFactory.buildPersisted(getTestName(info), envName, portal.getId());
```

Comment:
Let's move this into a new  portalFactory method like "buildPersistedWithEnvironments"